packages/components/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -146,7 +146,6 @@ export interface SimpleConfirmationModalProps
 
 export interface ComplexConfirmationModalProps
   extends BaseConfirmationModalProps {
-  readonly ref: Ref<ConfirmationModalRef>;
```

Comment:
I'll have to check the notes, but it seems like something around types and refs got stricter.

```

type ConfirmationModalProps =
  | (SimpleConfirmationModalProps & (StringMessage | ChildrenMessage))
  | (ComplexConfirmationModalProps & (StringMessage | ChildrenMessage));

export const ConfirmationModal = forwardRef(function ConfirmationModalInternal(
  {...},
  ref: Ref<ConfirmationModalRef>,
```

this is being used in a forwardRef, so it makes sense to me that the `ref` definition shouldn't be here, it should be  on the `forwardRef` function call

Reply:
this was causing `npm i` to fail on 0.74.0+

Reply:
having dealt with more type problems in `site` something definitely bumped the version of `@types/react` that was being used causing all sorts of TS errors